pkgs/tools/misc/lesspipe/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, perl }:
+
+stdenv.mkDerivation rec {
+  name = "lesspipe";
```

Comment:
To clarify: the use of `version` as a separate attribute is only to ease updating, it currently has no meaning (that may change).